app/utils/callback/lava.py

```diff
@@ -68,6 +68,7 @@
     models.FASTBOOT_KEY: "platform.fastboot",
     models.INITRD_KEY: "job.initrd_url",
     models.BOARD_KEY: "platform.name",
+    models.PLAN_KEY: "test.plan",
```

Comment:
So this is a new metadata field passed by LAVA ? Could you elaborate on what this test.plan is ? Is it just a metadata field that should be in the LAVA job description or is it "created" by LAVA one doing some testing ?

Reply:
This is already passed by LAVA in the job meta-data, it is in fact set when the LAVA job is submitted.  Typically, a kernelci.org job will have this set to "boot".  As we're adding more tests, other test plans such as "kselftest" are being used.

Reply:
OK thanks for the info